frontend/src/components/ShootMessages/ShootMessages.vue

```diff
@@ -164,7 +161,7 @@ export default {
             purposeText: this.shootPurpose || '',
             shootName: this.shootName || '',
             shootNamespace: this.shootNamespace || '',
-            showNavigationLink: this.canPatchShoots
+            showNavigationLink: this.canPatchShoots && !isEmpty(this.shootName) && !isEmpty(this.shootNamespace)
```

Comment:
Sometimes `shootNamespace` and `shootName` is empty. Rendering the link leads to an error in this case.